source/databases.txt

```diff
@@ -35,6 +35,35 @@ desired database in the left-hand navigation.
 You can also :ref:`create <compass-create-database>` or
 :ref:`drop databases <compass-drop-database>` from this view.
 
+Required Roles
+--------------
+
+The following table describes the roles required to perform various actions to 
+a database on an Atlas cluster:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 30 40
+
+   * - Action
+     - Required Roles
+
+   * - Create Databases
+     - One of the following roles:
+       
+       - :authrole:`Project Owner` or :authrole:`Organization Owner`
+       - :authrole:`Project Data Access Admin`
+       - :authrole:`Project Data Access Read/Write`
```

Comment:
Do organization and project owner roles imply that these users are able to do the same things as the other roles? Would it be enough to say "at least" `Project Data Access Read/Write` and `Project Data Access Admin`? (Genuine question, I'm not sure if there's a standard we need to follow here to also include the owner roles)

This applies to all other relevant places in the PR as well